### ISSUE TITLE
[Alpha][Bug] Environment variables are lost after app update

### ISSUE THREAD

**Issue by Khapra** [IceWhaleTech/CasaOS#915]
### What happened?

Updated to v0.4.2 this morning, then tried to update individual containers. Noticed the environment variables for all of them were gone. Not sure if the CasaOS v0.4.2 did that or individually updating container.

**Comment by tigerinus** [comment]
@Khapra which specific app did you update?

**Comment by Khapra** [comment]
@tigerinus i updated 13 apps, all of them ended up with the same problem. Most of them are from the CasaOS App Store. To name a few, radarr, sonarr, lidarr, readarr, Jellyfin, bazarr, embystat, adguardhome and qbittorent.

**Comment by Khapra** [comment]
@tigerinus I can confirm that individually updating the apps removed the environment variables. While updating, Prowlarr was already updated, just checked the settings and the environment variables are not removed, still the old ones.

**Comment by tigerinus** [comment]
Thanks for reporting the issue. Looking into it.

**Comment by raller1028** [comment]
v0.4.2-hotfix1 has been fixed this problem, you can try to update again. If still have problem, please reopen.
update command: `curl -fsSL https://get.casaos.io/update | sudo bash`

**Comment by tigerinus** [comment]
@Khapra - We've fixed this issue in `v0.4.2-1`.

If you are already running `v0.4.2`, please manually run the following commands to patch:

```
wget https://github.com/IceWhaleTech/CasaOS-AppManagement/releases/download/v0.4.2-1/linux-amd64-casaos-app-management-v0.4.2-1.tar.gz -O app-management.tar.gz
tar zxvf app-management.tar.gz
sudo systemctl stop casaos-app-management
sudo cp -rv build/sysroot/usr/bin/casaos-app-management /usr/bin/casaos-app-management
sudo systemctl start casaos-app-management
rm app-management.tar.gz
```

This patch will make sure this issue does not happen in future.

For the containers that already lost the environment variables, unfortunately you will have to either manually add them or re-install. Sorry about that.
